goosebit/ui/bff/devices/requests.py

```diff
@@ -10,3 +10,4 @@ class DevicesPatchRequest(BaseModel):
     pinned: bool | None = None
     feed: str | None = None
     force_update: bool | None = None
+    auth_token: str | None = None
```

Comment:
Should it be added to the bff even though the UI does not use it? In a way it is dead code like this.

Reply:
Maybe?  AFAIU there may be plans to show it on the UI, but fine to leave out.

Reply:
Let's add it when we get there. Otherwise: dead code.

Reply:
Used in tests.